inception/inception-experimental-editor/src/main/java/de/tudarmstadt/ukp/inception/experimental/editor/resources/AnnotationExperienceAPIBasicEditor.js

```diff
@@ -0,0 +1,1644 @@
+/*
```

Comment:
Considering that different parts of the file are annotated with what appears to be source file names, I assume that this file here is a pre-compiled version. The Maven build process should build these files and they should not normally not be checked in. Even if they are checked in as a convenience, the Maven build process must allow building the files and they should be built in a normal "mvn clean install" invocation.